.vitepress/config.mts

```diff
@@ -8,43 +8,73 @@ import ViteUnoCSS from 'unocss/vite'
 import { mermaid } from './md-mermaid'
 import { katex } from '@mdit/plugin-katex'
 
-function getGuideSidebar(): DefaultTheme.SidebarItem[] {
+function nav(): DefaultTheme.NavItem[] {
   return [
     {
-      text: 'Getting started',
+      text: 'Get Started',
+      link: '/get-started/index',
+      activeMatch: '/get-started/',
+    },
+    {
+      text: 'Developer Guide',
+      link: '/guide/introduction',
+      activeMatch: '/guide/',
+    },
+    {
+      text: 'API Reference',
+      link: '/api/index',
+      activeMatch: '/api/',
+    },
+    {
+      text: 'Troubleshooting',
+      link: '/guide/troubleshooting/overview',
+      activeMatch: '/guide/troubleshooting/',
+    },
+  ]
+}
+
+function sidebarAPI(): DefaultTheme.SidebarItem[] {
+  return [
+    {
+      text: 'API',
       items: [
         {
-          text: 'How Iroha works',
-          link: '/',
+          text: 'Specification',
+          link: '/api/index.md',
```

Comment:
Actually, I would rename the whole directory to `/reference/` instead of `/api/`